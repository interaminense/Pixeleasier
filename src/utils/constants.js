export const prefix = 'pixeleasier';

export const classes = {
	container: `${prefix}-container`,
	draw: `${prefix}-draw`,
	row: `${prefix}-row`,
	pixel: `${prefix}-pixel`,
	css: `${prefix}-css`
};