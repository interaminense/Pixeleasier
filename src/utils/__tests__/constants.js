import {prefix, classes} from '../constants';

describe('prefix', () => {
	it('should return the prefix name "pixeleasier"', () => {
		expect(prefix).toEqual('pixeleasier');
		expect(prefix).toMatchSnapshot();
	});
});

describe('classes', () => {
	const {container, draw, row, pixel, css} = classes;

	it('should return the container classname', () => {
		expect(container).toEqual(`${prefix}-container`);
		expect(container).toMatchSnapshot();
	});

	it('should return the draw classname', () => {
		expect(draw).toEqual(`${prefix}-draw`);
		expect(draw).toMatchSnapshot();
	});

	it('should return the row classname', () => {
		expect(row).toEqual(`${prefix}-row`);
		expect(row).toMatchSnapshot();
	});

	it('should return the pixel classname', () => {
		expect(pixel).toEqual(`${prefix}-pixel`);
		expect(pixel).toMatchSnapshot();
	});

	it('should return the css classname', () => {
		expect(css).toEqual(`${prefix}-css`);
		expect(css).toMatchSnapshot();
	});
});