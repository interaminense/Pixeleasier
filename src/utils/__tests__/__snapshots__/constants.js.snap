// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`classes should return the container classname 1`] = `"pixeleasier-container"`;

exports[`classes should return the css classname 1`] = `"pixeleasier-css"`;

exports[`classes should return the draw classname 1`] = `"pixeleasier-draw"`;

exports[`classes should return the pixel classname 1`] = `"pixeleasier-pixel"`;

exports[`classes should return the row classname 1`] = `"pixeleasier-row"`;

exports[`prefix should return the prefix name "pixeleasier" 1`] = `"pixeleasier"`;
